File: Operadores_relacionales.py
print ("Ingrese el numero de casos que desea evaluar", " ")
print ("1. Uno", " ")
print ("2. Dos", " ")
print ("3. Tres", " ")
casos = int (input ("Respuesta: "))
print (" ")

def caso1 (n1, n2):
    if n1 > n2:
        return "es mayor que"
    
    elif n1 == n2:
        return "es igual a"

    elif n1 < n2:
        return "es menor que" 

def caso2 (n3, n4):
    if n3 > n4:
        return "es mayor que"
    
    elif n3 == n4:
        return "es igual a"

    elif n3 < n4:
        return "es menor que"

def caso3 (n5, n6):
    if n5 > n6:
        return "es mayor que"
    
    elif n5 == n6:
        return "es igual a"

    elif n5 < n6:
        return "es menor que" 

if casos == 1:
    print ("Caso 1")
    n1 = int (input ("Ingrese el primer numero: "))
    n2 = int (input ("Ingrese el segundo numero: "))
    print (" ")
    print (n1, caso1(n1, n2), n2)

elif casos == 2:
    print ("Caso 1")
    n1 = int (input ("Ingrese el primer numero: "))
    n2 = int (input ("Ingrese el segundo numero: "))
    print (" ")

    print ("Caso 2")
    n3 = int (input ("Ingrese el primer numero: "))
    n4 = int (input ("Ingrese el segundo numero: "))
    print (" ")
    print (n1, caso1(n1, n2), n2)
    print (n3, caso2(n3, n4), n4)


elif casos == 3:
    print ("Caso 1")
    n1 = int (input ("Ingrese el primer numero: "))
    n2 = int (input ("Ingrese el segundo numero: "))
    print (" ")

    print ("Caso 2")
    n3 = int (input ("Ingrese el primer numero: "))
    n4 = int (input ("Ingrese el segundo numero: "))
    print (" ")

    print ("Caso 3")
    n5 = int (input ("Ingrese el primer numero: "))
    n6 = int (input ("Ingrese el segundo numero: "))
    print (" ")
    print (n1, caso1(n1, n2), n2)
    print (n3, caso2(n3, n4), n4)
    print (n5, caso3(n5, n6), n6)

else: print ("Valor ingresado es incorrecto")

File: Operadores_relacionales.cpp
#include <iostream>
#include <cmath>

using namespace std;

int n1, n2, n3, n4, n5, n6; 
char caso1 (int n1, int n2);
char caso2 (int n3, int n4);
char caso3 (int n5, int n6);

int main (){
    int casos;
    cout << "Ingrese el numero de casos que desea evaluar:" << endl << "1. Uno" << endl << "2. Dos" << endl << "3. Tres" << endl << "Respuesta: ";
    cin >> casos; cout << endl;

    switch (casos){
        case 1:
        cout << "Caso 1" << endl;
        cout << "Ingrese un numero entero: "; cin >> n1;
        cout << "Ingrese el segundo numero entero: "; cin >> n2; cout << endl;
        cout << n1 << caso1 (n1, n2) << n2;
        break;

        case 2:
        cout << "Caso 1" << endl;
        cout << "Ingrese un numero entero: "; cin >> n1;
        cout << "Ingrese el segundo numero entero: "; cin >> n2; cout << endl;

        cout << "Caso 2" << endl;
        cout << "Ingrese un numero entero: "; cin >> n3;
        cout << "Ingrese el segundo numero entero: "; cin >> n4; cout << endl;
        cout << n1 << caso1 (n1, n2) << n2 << endl;
        cout << n3 << caso2 (n3, n4) << n4;
        break;

        case 3:
        cout << "Caso 1" << endl;
        cout << "Ingrese un numero entero: "; cin >> n1;
        cout << "Ingrese el segundo numero entero: "; cin >> n2; cout << endl;

        cout << "Caso 2" << endl;
        cout << "Ingrese un numero entero: "; cin >> n3;
        cout << "Ingrese el segundo numero entero: "; cin >> n4; cout << endl;

        cout << "Caso 3" << endl;
        cout << "Ingrese un numero entero: "; cin >> n5;
        cout << "Ingrese el segundo numero entero: "; cin >> n6; cout << endl;
        cout << n1 << caso1 (n1, n2) << n2 << endl;
        cout << n3 << caso2 (n3, n4) << n4 << endl;
        cout << n5 << caso1 (n5, n6) << n6;
        break;

    }

    return 0;
}

char caso1 (int n1, int n2){
    if (n1 > n2){
        cout << " es mayor que";
    }
    else if (n1 < n2){
        cout << " es menor que";
    }
    else if (n1 = n2){
        cout << " es igual a";
    }
}

char caso2 (int n3, int n4){
    if (n3 > n4){
        cout << " es mayor que";
    }
    else if (n3 < n4){
        cout << " es menor que";
    }
    else if (n3 = n4){
        cout << " es igual a";
    }
}

char caso3 (int n5, int n6){
    if (n5 > n6){
        cout << " es mayor que";
    }
    else if (n5 < n6){
        cout << " es menor que";
    }
    else if (n5 = n6){
        cout << " es igual a";
    }
}